build/common.ps1

```diff
@@ -14,6 +14,8 @@ $Artifacts = Join-Path $NuGetClientRoot artifacts
 $Nupkgs = Join-Path $Artifacts nupkgs
 $ReleaseNupkgs = Join-Path $Artifacts ReleaseNupkgs
 $ConfigureJson = Join-Path $Artifacts configure.json
+$ILMergeOutputDir = Join-Path $Artifacts "VS14"
```

Comment:
Why VS14?

Reply:
It's also never used :)

Reply:
it was already there in earlier PR so I kept it. But will get rid of it if not used.